.env.example

```
@@ -5,12 +5,13 @@
-
-# Backend API (if using custom backend for AI)
+# AI backend endpoint
+# Do not place provider secrets in EXPO_PUBLIC_* variables.
+# Keep OpenAI / Anthropic / other provider API keys on the backend only.
 EXPO_PUBLIC_AI_API_URL=https://api.marketingtool.pro/api/generate
+
+# App Store / Play Store reviewer bypass
+# Never hardcode reviewer credentials in source control.
+# Configure reviewer-only values through secure environment or EAS secrets.
+REVIEWER_PHONE=
+REVIEWER_OTP=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REVIEWER_OTP key should go before the REVIEWER_PHONE key

(UnorderedKey)